src/improvements/tasks/MultisigTask.sol

```diff
@@ -225,72 +261,23 @@ abstract contract MultisigTask is Test, Script {
     /// @notice Sets the address registry, initializes the task.
     /// @param taskConfigFilePath The path to the task configuration file.
     function _taskSetup(string memory taskConfigFilePath) internal {
-        AddressRegistry _addrRegistry = new AddressRegistry(taskConfigFilePath);
-
-        _templateSetup(taskConfigFilePath);
-
-        _setMulticallAddress();
-
-        // set the task config
-        require(
-            bytes(config.safeAddressString).length == 0 && address(addrRegistry) == address(0x0),
-            "MultisigTask: already initialized"
-        );
-        require(
-            block.chainid == getChain("mainnet").chainId || block.chainid == getChain("sepolia").chainId,
-            string.concat("Unsupported network: ", vm.toString(block.chainid))
-        );
+        require(bytes(config.safeAddressString).length == 0, "MultisigTask: already initialized");
 
         config.safeAddressString = safeAddressString();
         config.allowedStorageWriteAccesses = _taskStorageWrites();
         config.allowedStorageWriteAccesses.push(safeAddressString());
 
-        // set the AddressRegistry
-        addrRegistry = _addrRegistry;
+        IGnosisSafe _parentMultisig; // TODO parentMultisig should be of type IGnosisSafe
```

Comment:
Is this a todo to change the type of the state variable `parentMultisig` to have IGnosisSafe type?

Reply:
I'm going to move that into a separate file at the end, just to make diffs easier to read in the interim (https://github.com/ethereum-optimism/superchain-ops/pull/688#discussion_r1975709845)

And yea I do want to change that type, but it has a cascading effect through this file giving a large diff, so it's not urgent and I'll come back to that